.github/workflows/tests.yaml

```diff
@@ -11,16 +11,20 @@ name: "Tests"
 
 jobs:
   phpunit:
-    name: "PHPUnit"
+    name: "PHPUnit (${{ matrix.php-version }})"
```

Comment:
Changing the name of the job here so we do not have

- awkward job descriptions
- to change the names of the jobs required to pass